test/utils/block.ts

```diff
@@ -489,10 +490,37 @@ export const getCurrentEraStartBlock = async (api: ApiPromise): Promise<number>
 
     let apiAtPreviousEpochBlock = await api.at(await api.rpc.chain.getBlockHash(epochStartBlock - 1));
 
-    while (currentEra === (await apiAtPreviousEpochBlock.query.externalValidators.currentEra())) {
+    while (
+        currentEra.unwrap().index.toNumber() ===
+        (await apiAtPreviousEpochBlock.query.externalValidators.activeEra()).unwrap().index.toNumber()
+    ) {
         epochStartBlock = (await apiAtPreviousEpochBlock.query.babe.epochStart()).toJSON()[1];
         apiAtPreviousEpochBlock = await api.at(await api.rpc.chain.getBlockHash(epochStartBlock - 1));
     }
 
     return epochStartBlock;
 };
+
+// getBlockHash is not available unless we use the current api, hence the reason to use 2
```

Comment:
Maybe rename the block argument or add a description that the provided block is the block in the era we are looking at the first block, to make it a bit more clear.

Reply:
fixed